### PR TITLE
Add unit tests for JComponentRouterRulesStandardTest

### PR DESCRIPTION
**Pull request by wilsonge**
### Summary of Changes
Adds unit tests.

I've taken the com_content router and the sample data we already use for tests rather than attempt to build up something that's nested and has a relevent menu structure etc

### Testing Instructions
Hope travis stays green

### Documentation Changes Required
None

### CODE REVIEW

File: tests/unit/suites/libraries/cms/component/router/stubs/ComContentRouter.php [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+		parent::__construct($app, $menu);
+
+		$this->attachRule(new JComponentRouterRulesStandard($this));
+	}
+
+	/**
+	 * Method to get the segment(s) for a category
+	 *
+	 * @param   string  $id     ID of the category to retrieve the segments for
+	 * @param   array   $query  The request that is build right now
```
Comment (brianteeman) [inline tests/unit/suites/libraries/cms/component/router/stubs/ComContentRouter.php]: built

File: tests/unit/suites/libraries/cms/component/router/stubs/ComContentRouter.php [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+		}
+
+		return array();
+	}
+
+	/**
+	 * Method to get the segment(s) for a category
+	 *
+	 * @param   string  $id     ID of the category to retrieve the segments for
+	 * @param   array   $query  The request that is build right now
```
Comment (brianteeman) [inline tests/unit/suites/libraries/cms/component/router/stubs/ComContentRouter.php]: built

File: tests/unit/suites/libraries/cms/component/router/stubs/ComContentRouter.php [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+	public function getCategoriesSegment($id, $query)
+	{
+		return $this->getCategorySegment($id, $query);
+	}
+
+	/**
+	 * Method to get the segment(s) for an article
+	 *
+	 * @param   string  $id     ID of the article to retrieve the segments for
+	 * @param   array   $query  The request that is build right now
```
Comment (brianteeman) [inline tests/unit/suites/libraries/cms/component/router/stubs/ComContentRouter.php]: built

Comment (wilsonge) [inline tests/unit/suites/libraries/cms/component/router/stubs/ComContentRouter.php]: Thanks! As this is a direct copy of the com_content router I guess this needs changing there (although out of scope for this PR - dunno if you want to put something in for that?)

**Pull request merged**